plugins/analytics-module-ga/src/apis/implementations/AnalyticsApi/GoogleAnalytics.ts

```diff
@@ -151,4 +183,67 @@ export class GoogleAnalytics implements AnalyticsApi {
 
     return customDimensionsMetrics;
   }
+
+  /**
+   * Sets the GA userId, based on the `userEntityRef` set on the backstage
+   * identity loaded from a given Backstage Identity API instance. Because
+   * Google forbids sending any PII (including on the userId field), we hash
+   * the namespace and name of the `userEntityRef` on behalf of integrators:
+   *
+   * - With value `User:default/name`, userId becomes `sha256(default/name)`
```

Comment:
In pretty much all other places we tell people consistently to treat the entity refs as opaque strings. I am surprised to see the kind left out here; feels like it should be left in

Reply:
Could be!  I guess the question is, would doing so make the "customize your hash" approach more confusing than it already is?  e.g.

- If you pass in `PrivateUser:foo/bar`, than we will set `userId` to `foo/bar` (because no one would ever want to also capture this weird, hacky string thing)
- But If you pass in `User:foo/bar`, then we will set `userId` to `sha256(User:foo/bar)`

My thinking was: because we're specifically capturing a User ID (and never any other kind of entity), it seems fine to drop it.